Choose best RXCUI: SCD, SBC, GPCK, BPCK, or other (#4423)

dckc opened the pull request.

dckc committed:
Choose best RXCUI: SCD, SBC, GPCK, BPCK, or other (#4423)

dckc committed:
explain (and cite source for ) PARTITION BY logic

mprittie commented on Oracle/pcornet_mapping.sql:
```diff
@@ -363,26 +363,93 @@ and ht.m_exclusion_cd is null
+    select distinct clarity_med_id medication_id, med_map.rxcui
+         , coalesce(spec_order, '9) HERON mapping misc.') spec_order
+    from "&&i2b2_etl_schema".clarity_med_id_to_rxcui@id med_map
+    join cui_pref on cui_pref.rxcui = med_map.rxcui
```
Looks like this should be a `left join`

mprittie commented on Oracle/pcornet_mapping.sql:
```diff
@@ -363,26 +363,93 @@ and ht.m_exclusion_cd is null
+from all_med
+left join med_map_best on all_med.concept_cd = 'KUH|MEDICATION_ID:' || med_map_best.medication_id
+;
+;
```
Unnecessary semi-colon

dckc committed:
Oops! the 9) misc case needs a left join

Thanks, Mike, for review.

Elaborate "Eyeball it" queries and test-sketches a bit.
